### ISSUE TITLE
Add docs for Python bindings, based on the docstrings

### ISSUE THREAD

**Issue by MichaelClerx** [cellml/libcellml#207]
~~There doesn't seem to be a way to do this automatically~~

~~www.swig.org/Doc1.3/Python.html#Python_nn65~~

Once we have docstrings (see #208) we could generate Sphinx documentation without too much hassle, and stick it online somewhere.

Thoughts? @jonc125 @hsorby @agarny @nickerso

**Comment by jonc125** [comment]
Yep, sounds sensible. Presumably could be integrated into the readthedocs build for the C++ sources?
